daemon/protocol/service.go

```diff
@@ -81,3 +83,60 @@ func DriverStates() *DriverStatesResponse {
 
 	return resp
 }
+
+//proteus:generate
+type InstallDriverRequest struct {
+	// Language supported by the driver being installed.
+	Language string
+	// ImageReference name of the image to be installed in the following format:
+	// `[transport://]name[:tag]` if transport is missing `docker` or tag is
```

Comment:
* s/ImageReference name/ImageReference is the name/
* s/if transport is missing `docker` or tag is missing `lastest` are assumed./Default values are: `docker` for transport and `latest` for tag/